public/js/modals.js

```diff
@@ -21,7 +22,7 @@ export function renderModal(data, close = true) {
       .addElems('button', 'close')
       .on('click', function () {
         if (typeof close === 'function') {
-          resolve(close());
```

Comment:
not sure where the resolves were supposed to come from. were those functions returning Promises at one point?

Reply:
more below